CHANGELOG.rst

```diff
@@ -3,6 +3,11 @@
 0.62.0
 ------
 
+Session secrets make it possible to connect to protected resources, such as databases or external
+compute systems, from a Renku session in a standardized and shareable manner. Collaborating with
+secrets is easy too: configure a single session secret slot to ensure that the secret shows up
+the same way for everyone, and each person enters their own value.
```

Comment:
Do we need to make it clearer that the secrets themselves are not shared, just the configuration?

Reply:
tagging @lokijuhy who wrote the release text.

Reply:
Oopsie, I had the old copy. Now updated with 5685b710ceb6740d2b7377145ff48a133b91b918.